src/sentry/static/sentry/app/utils/tokenizeSearch.tsx

```diff
@@ -28,24 +28,30 @@ export function tokenizeSearch(query: string) {
   };
 
   for (const token of tokens) {
-    const tokenChars = Array.from(token);
     let tokenState: 'query' | 'tags' = 'query';
 
-    tokenChars.forEach((char, idx) => {
-      const nextChar = tokenChars.length > idx ? tokenChars[idx + 1] : null;
-
-      if (idx === 0 && [':', ' '].includes(char)) {
-        return;
-      }
+    // Traverse the token and determine if it is a tag
```

Comment:
@markstory i thought this is a 1:1 translation from the python implementation. Does this mean the python implementation is buggy as well?

Reply:
I've updated this code to be laid out the same as the python version.

Reply:
@EvanPurkhiser

Reply:
oops :)